src/test/java/co/com/api/test/modelBody/DataModel.java

```diff
@@ -0,0 +1,32 @@
+package co.com.api.test.modelBody;
+
+public class DataModel {
```

Comment:
use a more meanly name, I suggest PersonDto

also model is not a good name for this package because model is associated with database in this case is a Dto because is a HTTP request

Reply:
Done